index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+</head>
+<body>
+  <meta charset="UTF-8">
+  <title>Weather Project</title>
+  <link href="http://maxcdn.bootstrapcdn.com/bootstrap/3.2.0/css/bootstrap.min.css" rel="stylesheet">
+  <link rel="stylesheet" href="style.css">
+</head>
+<body>
+  <div class="row">
+    <div class="col-md-6 col-md-offset-3">
+
+      <div class="page-header">
+        <h1>Weather Project</h1>
+      </div>
+        <div class="form-group">
+          <input id="textbox" type="text"
+            class="form-control"
+            placeholder="Enter City Name Here"></input>
+        </div>
+        <button id="submit" class="search button">Search</button>
+      <div class="weather-container"></div>
+    </br>
+    
+      <div class="five-day-container"></div>
+    </div>
+  </div>
+  
+  <script src="main.js"></script>
+  <script src="style.css"></script>

```

Comment:
remove this line cause its trying to add the css as js and it causes a console error